### PR TITLE
Dev/release api

### PR DESCRIPTION
**Pull request by shivamgoyal7**
### Functionality:
added release API data in the footer

### Solution:
added release API data in the footer.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

**Review by viv9k — APPROVED** [review]
Ship it

File: src/app/footer/footer.component.html [hunk 1]
```diff
@@ -18,6 +18,18 @@
...
+                Id: {{id}}
+            </div>
+            <div class="row">
+               Created_at: {{created_at}}
+            </div>
+            <div class="row">
+                Published_at: {{published_at}}
+            </div>
         </div>
         <div class="col text-right"><a>Build:Worktrolly 2.3.0</a></div>
```
Comment (viv9k) [inline src/app/footer/footer.component.html:34]: The UI is not good only change 2.3.0 to tag_name

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/footer/footer.component.html (hunk 1 above)
Comment (shivamgoyal7) [reply src/app/footer/footer.component.html]: its done

File: src/app/footer/footer.component.html [hunk 2]
```diff
@@ -19,6 +19,6 @@
                 </div>
             </div>
         </div>
-        <div class="col text-right"><a>Build:Worktrolly 2.3.0</a></div>
+        <div class="col text-right"><a>Build:Worktrolly {{tag_name}}</a></div>
```
Comment (viv9k) [inline src/app/footer/footer.component.html:22]: Remove worktrolly

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**